README.md

```diff
@@ -1,2 +1,45 @@
 # Hello-Microverse
-This is a trial code review repo
+This is a trial code review repo which is about showing Hello Microverse as a header.
+
+![screenshot](./app_screenshot.png)
+
+## Built With
+
+- HTML
+- CSS
+- Webhint and Stylelint
+
+## Live Demo
+
+N/A
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Open the terminal
+- Run `git clone https://github.com/MahmoudBakr23/Hello-Microverse.git`
+- Then run `git checkout feature`
+- Now you can open the `index.html` file in your browser
+
+## Authors
+
+👤 **Mahmoud Bakr**
+
+- GitHub: [@githubhandle](https://github.com/MahmoudBakr23)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
- [ ] This LinkedIn link is broken. It will be good to update it with the actual link to your linked-in account. This will improve the professional level of your project.